CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ the detailed section referring to by linking pull requests or issues.
 #### Minor
 
 - Implemented markdown for asset descriptions
+- Implemented list view for Broker UI
```

Comment:
![grafik](https://github.com/sovity/edc-ui/assets/30114507/c3af4aa4-45ea-4645-9e7a-07f80a26c797)

I overlooked the sort box first, but I think you found a good solution for how fat the Angular Material field is